internal/application/filesystem.go

```diff
@@ -648,7 +658,7 @@ func NewFileSystem() IApplication {
 	for _, application := range applications {
 		artifacts := fileSystem.GetTargetArtifactsList(application.Name)
 		for _, artifact := range artifacts {
-			err := fileSystem.Translate(application.Name, artifact, "")
+			err := fileSystem.Translate(application.Name, artifact, "", "")
```

Comment:
Can we store the debug level in which it was started in say the qa file, and reuse the debug level?

Reply:
Sure, I will check this. Currently I have replaced the `""` with `false` so that the build doesn't fail.

Reply:
Yes, I have updated the code to store the debug level in the QA file. So that, Move2Kube reuses the stored debug level after restart.